supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -1047,14 +1068,25 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 [discrete]
 [[rpm]]
 ==== image:images/yes.png[yes] RPM (noun)
-*Description*: _RPM_ is the recursive initialism for the _RPM Package Manager_. RPM manages files in the RPM format, known as RPM packages. RPM packages are known informally as `rpm` files, but this informal usage is not used in Red Hat documentation to avoid confusion with the command name. Files in RPM format are referred to as "RPM packages".
+*Description*: _RPM_ is the recursive initialism for the _RPM Package Manager_. RPM manages files in the RPM format. Those files are referred to as "RPM packages". Note that even though RPM packages are known informally as `rpm` files, this informal term must not be used in Red Hat documentation to avoid confusion with the command name.
```

Comment:
Doesn't 'RPM' stand for 'Red Hat Package Manager'? You all know more about this than I do, but I am doubly confused after reading the description.

Reply:
@IngridT1 - You would've been correct before 17 Jan 2000: https://rpm.org/timeline.html. That's when they changed the meaning from "Red Hat Package Manager" to a recursive acronyn of "RPM Package Manager".